.env.example

```diff
@@ -13,3 +13,7 @@ MAILGUN_RECIPIENT_OVERRIDE=
 BOOTCAMP_ADMISSION_BASE_URL=
 BOOTCAMP_ADMISSION_KEY=
 BOOTCAMP_ECOMMERCE_BASE_URL=
+FLUIDREVIEW_CLIENT_ID=
+FLUIDREVIEW_CLIENT_SECRET=
+FLUIDREVIEW_ACCESS_TOKEN=
+FLUIDREVIEW_REFRESH_TOKEN=
```

Comment:
Can you add `FLUIDREVIEW_BASE_URL` here too?

Reply:
Sure